conf/scripts/generate-saml-cert.sh

```
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+OUTDIR=./
+TMP=`mktemp -t`
+trap "rm $TMP* 2>/dev/null" 0
```

Shellcheck (0.10.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)